### PR TITLE
NIX in action documentation -- Relacs

### PR DESCRIPTION
**Pull request by jgrewe**
From my point of view, this should be enough for the relacs use-case. I'm sure, there is room for improvements. Please feel free to comment.

### CODE REVIEW

File: docs/relacs.rst [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+It is one use-case showing how to integrate automated data storage and
+data annotation into the recording process.
+
+Relacs overview
+---------------
+
+In the `Neuroethology group <www.neuroetho.uni-tuebingen.de>`__ relacs
+is used to record the activity of single neurons during
+electrophysiological experiments. It controls the experiment, presents
+stimuli, does some online analysis and stores the data to disk. It
```
Comment (achilleas-k) [inline docs/relacs.rst]: I would say "performs some online analysis".  Sounds nicer than a general "does".

File: docs/relacs.rst [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+requires a lot of flexibility and the configurability leads to the
+situation that options and settings are to some extent not
+foreseeable.
+
+.. figure:: ./images/relacs_ephys.png
+   :alt: relacs user interface
+   :align: left
+   :figwidth: 100 %
+
+   **Figure 1:** The relacs user interface. The ui is divided into four panels. Top
```
Comment (achilleas-k) [inline docs/relacs.rst]:
```suggestion
   **Figure 1:** The relacs user interface. The UI is divided into four panels. Top
```

File: docs/relacs.rst [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+| **Figure 2** | **Figure 3** |
++--------------+--------------+
+| |options|    |     |dialog| |
++--------------+--------------+
+
+
+A recording session
+-------------------
+
+Once the recording of a neuron has been established and we are ready
```
Comment (achilleas-k) [inline docs/relacs.rst]:
```suggestion
Once the recording of a neuron has been established, we are ready
```

File: docs/relacs.rst [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+to start running the actual experiment.  Either pre-defined macros are
+used to control the experiments or the experimenter starts research
+protocols (RePros) manually.
+
+When recording the data time is not necessarily real time. That is,
```
Comment (achilleas-k) [inline docs/relacs.rst]:
```suggestion
When recording the data, time is not necessarily real time. That is,
```

File: docs/relacs.rst [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+the data is only recorded when an active research protocol (RePro)
+sets the acquisition active and thus the data time is only runs when
```
Comment (achilleas-k) [inline docs/relacs.rst]: "is only runs"

Not sure if it should  be "is only run" or "only runs".

File: docs/relacs.rst [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+   **Figure 4:** A recording session becomes active once the acquisition is switched
+   on (red bar). Data acquisition, however, data is stored in the NIX
+   file, when the currently running RePros demands it. In this example
+   session there are two RePros active, the so called "Baseline"
+   (green) and the "fi-curve" (blue). Within the time the fi-curve
+   RePro is active three stimuli are presented (light blue). For each
+   stimulus presentation the fi-curve RePro applies a stimulus with a
+   different intensity (contrast, blue numbers).
+
+Data storage uding NIX
```
Comment (achilleas-k) [inline docs/relacs.rst]:
```suggestion
Data storage using NIX
```

**Comment by achilleas-k** [comment]
Some typos noted :)

**Comment by gicmo** [comment]
Very nice. The only thing I am a tiny bit concerned with is adding a blob of 11MB. Maybe we can put that file somewhere and have the python script download it, if it is not there? That way you could also call the python script from any location (currently the path is hardcoded with `../examples/`.

**Comment by achilleas-k** [comment]
Agreed on the 11MB NIX file.  Kinds sucks to have people download a sample file that's used for documentation when all they want is the code.

**Pull request merged**